src/entity/ReputationEvent.ts

```diff
@@ -15,6 +15,8 @@ export enum ReputationReason {
   PostReportConfirmed = 'post_report_confirmed',
   SourceRequestApproved = 'source_request_approved',
   CommentDownvoted = 'comment_downvoted',
+  StreakRecover = 'streak_recover',
+  StreakRecoverForFree = 'streak_recover_for_free',
```

Comment:
I think it might be better to say `StreakFirstRecovery`. Not a blocker though, I think it is more explicit.

Reply:
agreed